packages/component-library/draft/Kaizen/Collapsible/Collapsible.tsx

```diff
@@ -11,125 +12,163 @@ const chevronUp = require("@cultureamp/kaizen-component-library/icons/chevron-up
 const chevronDown = require("@cultureamp/kaizen-component-library/icons/chevron-down.icon.svg")
   .default
 
-export type Props = {
-  id: string
-  children: JSX.Element | JSX.Element[] | string
-  title: string
-  renderHeader?: (title: string) => JSX.Element | JSX.Element[]
-  open?: boolean
-  group?: boolean
-  separated?: boolean
-  sticky?: Sticky
-  noSectionPadding?: boolean
-  automationId?: string
-  onToggle?: (open: boolean, id: string) => void
+export type Props =
+  | {
+      loading: true
+      separated?: boolean
+      group?: boolean
 
-  /* Will avoid rendering the content until required (especially important when you have queries inside sections). Removes animation. */
-  lazyLoad?: boolean
+      /* This won't actually do anything in this context, but it's required to keep
+         the internalOpen state hook at the top of the function below */
+      open?: boolean
```

Comment:
The other option here is to remove this pattern and allow all props along with `loading`. Neither is ideal, I decided to go with this to stay consistent with `CollapsibleGroup`, which has more reason for this pattern because of the `collapsibles` prop that only applies on `loading`. I could go either way on this.